patches/747-mikrotik-extra-support.patch

```diff
@@ -552,44 +573,48 @@
 +#include "qcom-ipq4018-sxtsq-5-ac.dts"
 +
 +/ {
-+	compatible = "mikrotik,ldf-5acd";
++ compatible = "mikrotik,ldf-5acd";
 +	model = "MikroTik LDF 5 ac (RBLDFG-5acD)";
 +};
 --- a/target/linux/ipq40xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata
 +++ b/target/linux/ipq40xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata
-@@ -210,6 +210,8 @@
+@@ -210,6 +210,9 @@
  	mikrotik,hap-ac3 |\
  	mikrotik,hap-ac3-lte6-kit |\
  	mikrotik,sxtsq-5-ac |\
++ mikrotik,discg-5acd |\
```

Comment:
Still need to fix the whitespace here (and other places - please check the patch) where you have a space and not a tab.